.github/workflows/release.yml

```diff
@@ -75,7 +75,7 @@ jobs:
           # Allows pushing to registry on localhost:5000
           driver-opts: network=host
 
-      - name: Setup push rights to Docker Hub
+      - name: Setup push rights to Docker registry
```

Comment:
I like saying "container registry" for this as that is often how cloud providers refer to these for example. In my mind, they are actually "image registries" as I think of Dockerfiles being built into images, and started as containers etc. I've settled on container registry mentally though.

Reply:
I'll try and remember next time, otherwise feel free to remind me!